public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -60,6 +60,11 @@ async function fetchDashboard(
         dashDTO.meta.canStar = false;
         return dashDTO;
       }
+      case DashboardRoutes.Public: {
+        const dashDTO: DashboardDTO = await dashboardLoaderSrv.loadDashboard(args.urlType, args.urlSlug, args.urlUid);
+
+        return dashDTO;
```

Comment:
How does this work, surely we need to call a different API? the normal dashboard API will not be accessible to an un-authenticated user

Reply:
But maybe this is just temporary until that other PR is merged now that I think about it :)

Reply:
Yeah this is temporary until a later pr is merged that maps a public dashboard uid to a dashboard definition